EIPS/eip-erps/eip-erps.md

```diff
@@ -0,0 +1,194 @@
+
+
+## ERP Approval Process
+Approval of any ERP will require a mandatory minimum of 30-days between the time the ERP is submitted (or substantially modified) and adoption by clients, and ERPs affecting more funds or more addresses may prompt a considerably longer review period.  In practice, ERPs will often require more time because the will likely be bundled into other client upgrades that are on their own schedule.  However, the 30-day mandatory review period  guarantees a period for comment and auditing and prevents any last minute inclusions from being rushed into an upcoming upgrade.  
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
+
+- People or organizations who are directly affected by the issue (e.g. entities who will be refunded or were associated with the original incident)
+- People or organizations that have a demonstrable history within the Ethereum community that are unaffected by the issue but may serve as relatively impartial adjudicators
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to the this meta-EIP.  Community members in general are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
+
+The ERP author may indicate in the ERP comments when he/she believes there is sufficient support and request a review by an EIP editor (see EIP-1 for details).  The editor(s) will mark the ERP as “Accepted” or “Rejected”.  As with any EIP, a Rejected ERP may be re-submitted upon substantial improvement.  
+
+Once an ERP is marked as “Accepted” by an EIP Editor, the block at which the changes will be applied will be finalized based on coordination between the client teams and other pending releases.  Once the block is determined, a finalized version (updated to include the determined block) of the State Change Object will be published to the ERP.  Clients will bundle this final version (unmodified) with their respective applications.  
```

Comment:
What prevents the EIP Editor from being sued for marking/not marking "Accepted"?

Reply:
I guess, for this to work, all Ethereum users need to sign a document, saying, they will be following the decisions taken in the ERP approving process, no?

Reply:
> What prevents the EIP Editor from being sued for marking/not marking "Accepted"?

Certainly, even without this EIP, no one can claim that "no EIP Editor can ever be sued for anything".  In fact, I would be hard-pressed to fill in the blank in "no <insert role> can ever be sued for anything."

However, according to EIP-1 "The editors don't pass judgment on EIPs. We merely do the administrative & editorial part."  This would apply to ERPs as well.  Editors should expect ERPs to provide clear and convincing evidence as well as explicit feedback from others in the community (the second group mentioned) to indicate that they also find the evidence clear and convincing.  

Keep in mind that the ERP process does not apply to cases where there is a reasonable disagreement between directly affected parties.  It is **not** about determining "guilt" or "innocence" or making a value judgment.  It is specifically meant to addresses cases where ownership is clear (similar to #156 )

> I guess, for this to work, all Ethereum users need to sign a document, saying, they will be following the decisions taken in the ERP approving process

I don't think so, but I am not a lawyer.  Do all Ethereum users sign anything right now?